README.md

```diff
@@ -24,6 +24,15 @@ sensor:
     password: 'XXXX'
     station_id : '12345678-1234-1234-1234-123456789012'
     scan_interval: 60
+
+# create a template to access the data as "sensor.pv_outputpower"
```

Comment:
Cool. Maybe we should make clear that this is optional/extra configuration, but not required to make the component work.

Reply:
"example template" instead of "create a template" ?